grails-app/conf/Config.groovy

```diff
@@ -221,7 +221,8 @@ portal {
     siteHeader = "Open Access to Ocean Data"
     motdUrl = "https://static.emii.org.au/motd"
 
-    logo = "images/IMOS_logo.png"
+    logo = "images/AODN/AODN_logo.png"
+    secondaryLogo ="https://static.emii.org.au/images/logo/IMOS_logo-stacked3.png"
```

Comment:
I'm not reviewing the PR per se, but here you're missing a space.

But the more important thing is that this will become the default now, unless overriden. So either communicate it with IMAS (config changes on their side), or make the default logo blank so we can override it in our config.